lib/resty/kafka/client.lua

```diff
@@ -3,6 +3,7 @@
 
 local broker = require "resty.kafka.broker"
 local request = require "resty.kafka.request"
+local Errors = require "resty.kafka.errors"
```

Comment:
```suggestion
local errors = require "resty.kafka.errors"
```
style

Reply:
I have noticed that `Errors` in other files use this style.

https://github.com/doujiang24/lua-resty-kafka/blob/659f2b131a4d25c7524963d806974e342b76b601/lib/resty/kafka/producer.lua#L8

Reply:
I think `errrors` could be better.